### PR TITLE
dockerfile, makefile, license, zappr, travis

### PR DESCRIPTION
**Pull request by ideahitme**

### CODE REVIEW

**Comment by ideahitme** [comment]
will enable in zappr and travis once it is merged

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,54 @@
+.PHONY: clean check build.local build.linux build.osx build.docker build.push
+
+BINARY        ?= kube_ingress_aws_controller
```
Comment (Raffo) [inline Makefile]: Why the name with underscores? `kube-ingress-aws-controller` would be better.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM registry.opensource.zalan.do/stups/alpine:UPSTREAM
+MAINTAINER Team Teapot @ Zalando SE <team-teapot@zalando.de>
+
+# add scm-source
+ADD scm-source.json /
+
+# add binary
+ADD build/linux/kube_ingress_aws_controller /
+
+ENTRYPOINT ["/kube_ingress_aws_controller"]
```
Comment (Raffo) [inline Dockerfile]: For the name, obviously the same as below applies.

**Comment by linki** [comment]
👍

**Pull request merged**